### PR TITLE
Update csslint library to avoid blank rules

### PR DESCRIPTION
**Pull request by GordonDiggs**
/cc @codeclimate/review 

Relevant CSSLint commits:

* https://github.com/codeclimate/csslint/commit/151db723f44685821d2a63447060bceed7f89f00
* https://github.com/codeclimate/csslint/commit/2a53712c61710840c023978418c7b48e3f32ac64

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+FROM alpine:edge
 
 WORKDIR /usr/src/app
 COPY Gemfile /usr/src/app/
 COPY Gemfile.lock /usr/src/app/
 
-RUN npm install -g csslint && \
-    apk --update add ruby ruby-dev ruby-bundler less ruby-nokogiri build-base && \
+RUN apk --update add nodejs git ruby ruby-dev ruby-bundler less ruby-nokogiri build-base && \
+    npm install -g codeclimate/csslint.git#2a53712c61710840c023978418c7b48e3f32ac64 && \
```
Comment (pbrisbin) [inline Dockerfile]: Where was the `npm install` happening before?

Comment (GordonDiggs) [inline Dockerfile]: It was originally on the first line of the `RUN` command. I had to move it to the second because it require git now.

Comment (pbrisbin) [inline Dockerfile]: Bah, totally misread the diff.

**Comment by pbrisbin** [comment]
LGTM

**Pull request merged**